docs/installation/production/app_installation.rst

```diff
@@ -26,29 +26,29 @@ You will need to copy the source code of the app to the server. There are many m
 
     Substitute "username" for your GitHub username or organization and substitute "tethysapp-my_first_app" for the name of the repository with your app source code.
 
-4. Install the App
+3. Install the App
 ==================
 
-Execute the setup script (:file:`setup.py`) with the ``install`` command to make Python aware of the app and install any of its dependencies::
+Execute the install command in the app directory to make Python aware of the app and install any of its dependencies::
 
-    (tethys) $ cd cd $TETHYS_HOME/apps/tethysapp-my_first_app
-    (tethys) $ python setup.py install
+    (tethys) $ cd $TETHYS_HOME/apps/tethysapp-my_first_app
+    (tethys) $ tethys install
```

Comment:
Maybe add a note here to check the apps documentation for any additional installation configuration (e.g. adding something to `INSTALLED_APPS` in the `settings.py`).

Reply:
Resolved